charts/t8s-cluster/templates/management-cluster/clusterClass/clusterClass.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+metadata:
+  name: t8s-engine-v0.6.0
+  namespace: {{ $.Release.Namespace }}
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+spec:
+  controlPlane:
+    machineHealthCheck:
+      maxUnhealthy: 33%
+      nodeStartupTimeout: 10m
+      unhealthyConditions:
+        - status: Unknown
+          timeout: 600s
+          type: Ready
+        - status: 'False'
+          timeout: 600s
+          type: Ready
+    machineInfrastructure:
+      ref:
+        apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
+        kind: OpenStackMachineTemplate
+        name: {{ include "common.names.fullname" $ }}-control-plane-{{ include "t8s-cluster.clusterClass.openStackMachineTemplate.specHashOfControlPlane" (dict "context" $) }}
+    ref:
+      apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+      kind: KubeadmControlPlaneTemplate
+      name: {{ include "common.names.fullname" $ }}-{{ include "t8s-cluster.clusterClass.kubeadmControlPlaneTemplate.specHash" . }}
```

Comment:
Is the context correct here?

Reply:
What do you mean by `context`?

That it's missing? Yes, this function doesn't use parameters, therefore I didn't use that

But I guess that it would be less confusing if it would have a `context` parameter

Reply:
Nvm, this is needed for `.Files.Get`

I added a comment to explain this